bin/compile-requirements.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -exo pipefail
+
+export CUSTOM_COMPILE_COMMAND="make compile-requirements"
+
+# We need this installed, but we don't want it to live in the main requirements
+# We will need to periodically review this pinning
+pip install --upgrade "pip<22"  # Temporary until pip-tools is fixed.
+pip install --upgrade pip-tools==6.4.0  # Needs at least this version to build.
+pip install --upgrade pip-compile-multi
+
+pip-compile-multi --generate-hashes prod --generate-hashes dev
```

Comment:
Suggestionm, if you want: Bedrock now has an additional param that lets us add a custom header to explain how to rebuild deps, replacing the default which is misleading in this context

`--header=/app/bin/pip-compile-multi-header-message.txt`